web/packages/hovercards/src/core.ts

```diff
@@ -295,32 +366,227 @@ export default class Hovercards {
+		const profileColorEl = hovercardInner.querySelector< HTMLElement >( '.gravatar-hovercard__profile-color' );
+
+		if ( headerImage && headerImageEl ) {
+			headerImageEl.style.background = headerImage;
+		}
+
+		if ( backgroundColor && profileColorEl ) {
+			profileColorEl.style.background = backgroundColor;
+		}
+
+		hovercardInner.querySelector( '#contact-btn' )?.addEventListener( 'click', () => {
+			Hovercards._openDrawer( `#${ contactsDrawerId }`, hovercardInner );
+		} );
+
+		hovercardInner.querySelector( `#${ contactsDrawerId } .close-btn` )?.addEventListener( 'click', () => {
+			Hovercards._closeDrawer( `#${ contactsDrawerId }`, hovercardInner );
+		} );
+
+		hovercardInner.querySelector( `#${ contactsDrawerId } .backdrop` )?.addEventListener( 'click', () => {
+			Hovercards._closeDrawer( `#${ contactsDrawerId }`, hovercardInner );
+		} );
+
+		hovercardInner.querySelector( '#send-money-btn' )?.addEventListener( 'click', () => {
+			Hovercards._openDrawer( `#${ sendMoneyDrawerId }`, hovercardInner );
+		} );
+
+		hovercardInner.querySelector( `#${ sendMoneyDrawerId } .close-btn` )?.addEventListener( 'click', () => {
+			Hovercards._closeDrawer( `#${ sendMoneyDrawerId }`, hovercardInner );
+		} );
+
+		hovercardInner.querySelector( `#${ sendMoneyDrawerId } .backdrop` )?.addEventListener( 'click', () => {
+			Hovercards._closeDrawer( `#${ sendMoneyDrawerId }`, hovercardInner );
+		} );
 
 		return hovercard;
 	};
 
+	/**
+	 * Creates a hovercard drawer.
+	 *
+	 * @param {string} id        - The drawer id
+	 * @param {string} titleText - The title shown at the drawer's header
+	 * @param {string} content   - The drawer inner content
+	 * @return {string}          - The drawer HTML string
+	 */
+	private static _createDrawer( id: string, titleText: string, content: string ): string {
+		return `
+			<div id="${ id }" class="gravatar-hovercard__drawer">
+				<div class="gravatar-hovercard__drawer-backdrop backdrop"></div>
+				<div class="gravatar-hovercard__drawer-card">
+					<div class="gravatar-hovercard__drawer-header">
+						<h1 class="gravatar-hovercard__drawer-title">${ titleText }</h1>
+						<button class="gravatar-hovercard__drawer-close close-btn">
+							<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+								<path d="M12 13.0607L15.7123 16.773L16.773 15.7123L13.0607 12L16.773 8.28772L15.7123 7.22706L12 10.9394L8.28771 7.22705L7.22705 8.28771L10.9394 12L7.22706 15.7123L8.28772 16.773L12 13.0607Z" fill="#101517"></path>
+							</svg>
+						</button>
+					</div>
+					<ul class="gravatar-hovercard__drawer-items">
+						${ content }
+					</ul>
+				</div>
+			</div>
+		`;
+	}
+
+	/**
+	 * Opens a hovercard drawer.
+	 *
+	 * @param {string}  selector  - The drawer selector.
+	 * @param {Element} container - The container context to search for the drawer.
+	 * @return {void}
+	 */
+	private static _openDrawer( selector: string, container: Element ): void {
```

Comment:
Nit: Consider specifying a more precise element type:

```suggestion
	private static _openDrawer( selector: string, container: HTMLElement ): void {
```